tests/unit/acl.tcl

```diff
@@ -905,7 +905,7 @@ start_server {tags {"acl external:skip"}} {
     test {ACL load non-existing configured ACL file} {
        catch {r ACL load} err
        set err
-    } {*Redis instance is not configured to use an ACL file*}
+    } {*instance is not configured to use an ACL file*}
```

Comment:
i'm not sure
just suggest - uppercase for the first character: rename instance -> Instance

Reply:
This is a pattern in the test case. It's matching the error message "This instance is not configured to use an ACL file...." created in `acl.c`.

`*` is a wildcard character here.